overlord/servicestate/service_control_test.go

```diff
@@ -612,7 +628,14 @@ func (s *serviceControlSuite) TestStartAllServices(c *C) {
 
 	c.Assert(t.Status(), Equals, state.DoneStatus)
 
-	c.Check(s.sysctlArgs, DeepEquals, [][]string{
+	/* We don't care about the order of the is-enabled invocations, as long as
+	 * they all happen before the first invocation of "start" */
+	verifyUnsortedInvocations(c, s.sysctlArgs, 0, "is-enabled", []string{
+		"snap.test-snap.abc.service",
+		"snap.test-snap.bar.service",
+		"snap.test-snap.foo.service",
+	})
```

Comment:
Couldn't you just pass `s.sysctlArgs[:3]` and then simplify verifyUnsortedInvocations (it would require action match on all elements of the slice)? This seems to be implied by s.sysctlArgs[3:] below.

Reply:
mmm... yes, this indeed simplifies stuff. Thanks! (done)